packages/rke2-kube-proxy/charts/values.yaml

```
@@ -39,85 +39,6 @@ detectLocalMode:
 
 # A set of key=value pairs that describe feature gates for alpha/experimental features:
 featureGates: 
```

I think you will need to set this to an empty map otherwise you will get an error when the chart tries to iterate over a nil value.

```suggestion
featureGates: {}
```